README.md

```diff
@@ -2,4 +2,42 @@
 
 [![Build Status](https://travis-ci.org/coreos/jetcd.svg?branch=master)](https://travis-ci.org/coreos/jetcd)
 
-Java client for etcd v3.
+jetcd is a Java client library for the awesome [etcd](https://coreos.com/etcd/), a highly available distributed key-value store.
+
+### Etcd version support
+
+jetcd only supports newly etcd v3, for any v2 support, you can use Java bindings list [here](https://github.com/coreos/etcd/blob/master/Documentation/libraries-and-tools.md).
+
+### Download
```

Comment:
Rename this section to "Getting Started".
Then we talk about how to get from package manager, and some simple examples.
What do you think?